contracts/collections/sg721-base/src/contract.rs

```diff
@@ -207,6 +207,10 @@ where
             return Err(ContractError::Unauthorized {});
         }
 
+        if let Some(new_creator) = collection_msg.creator {
```

Comment:
would be nice to add addr_validate before storing it.

Reply:
LOL. You r right, ofc :).